formats/json/commonMain/src/kotlinx/serialization/json/Json.kt

```diff
@@ -170,6 +170,11 @@ public class JsonBuilder internal constructor(json: Json) {
      */
     public var encodeDefaults: Boolean = json.configuration.encodeDefaults
 
+    /**
+     * TODO xdfdx
+     */
+    public var omitNull: Boolean = json.configuration.omitNull
```

Comment:
We haven't decided yet on the name, right?

Reply:
Yes, we have not decided on a name.